lib/3.5/packages.cf

```diff
@@ -113,6 +138,25 @@ bundle common redhat_knowledge
       "rpm3_name_regex" string => "([^.]+).*";
       "rpm3_version_regex" string => "[^\s]\s+([^\s]+).*";
       "rpm3_arch_regex" string => "[^.]+\.([^\s]+).*";
```

Comment:
The whole section with "rpm*_regex" type of variables is now duplicated in both ``redhat_knowledge`` and ``rpm_knowledge``. I think we should stick to one, so we don't introduce subtle bugs later because one was being used instead of the other. The ``rpm_knowledge`` is the one that is used in 3.6 onwards, so it's probably more up to date.